services/src/main/java/org/apache/druid/server/AsyncQueryForwardingServlet.java

```diff
@@ -104,7 +104,7 @@ public class AsyncQueryForwardingServlet extends AsyncProxyServlet implements Qu
   private static final String PROPERTY_SQL_ENABLE = "druid.router.sql.enable";
   private static final String PROPERTY_SQL_ENABLE_DEFAULT = "false";
 
-  private static final int CANCELLATION_TIMEOUT_MILLIS = 500;
+  private static final int CANCELLATION_TIMEOUT_MILLIS = 1000;
```

Comment:
Nit: let's avoid magic number :smile:

```suggestion
  private static final int CANCELLATION_TIMEOUT_MILLIS = TimeUnit.SECONDS.toMillis(1);;
```

Reply:
Done.